.github/workflows/ci.yml

```diff
@@ -62,15 +62,14 @@ jobs:
       matrix:
         try-scenario:
           [
-            ember-lts-2.12,
-            ember-lts-2.18,
-            ember-lts-3.16,
-            ember-lts-3.20,
-            ember-lts-3.24,
             ember-lts-3.28,
             ember-lts-4.4,
             ember-lts-4.8,
             ember-lts-4.12,
+            ember-lts-5.3,
```

Comment:
so either 5.3 needs to just be removed or we bump it to 5.4. technically 5.4 isn't an LTS until 5.5 is released so maybe we just remove for now since the "normal" test (not with ember-try) is already testing 5.4 👍